willow/plugins/pillow.py

```diff
@@ -58,6 +58,30 @@ def crop(self, rect):
         return PillowImage(self.image.crop(rect))
 
     @Image.operation
+    def set_background_color_rgb(self, color):
+        if not self.has_alpha():
+            # Don't change image that doesn't have an alpha channel
+            return self
+
```

Comment:
Maybe raise a descriptive exception here if `color` isn't a three-tuple (or technically an array with at least three elements unless being explicit about the type) to avoid an uncaught exception at L73